presto-main/src/main/java/com/facebook/presto/split/SplitManager.java

```diff
@@ -69,6 +79,13 @@ public SplitSource getSplits(Session session, TableLayoutHandle layout)
         return splitSource;
     }
 
+    public SplitSource getSplits(Session session, TableLayoutHandle layout)
```

Comment:
Why do we need to have 2 different methods? I would rather keep only one and fix the tests.